pkg/controller/performanceprofile/components/profileinfo/profileinfo.go

```diff
@@ -0,0 +1,10 @@
+package profileinfo
+
+import performancev2 "github.com/openshift-kni/performance-addon-operators/api/v2"
+
+// PerformanceProfileInfo is a wrapper for PerformanceProfile that can hold extra data and configuration
+type PerformanceProfileInfo struct {
+	performancev2.PerformanceProfile
```

Comment:
please use a pointer

Reply:
For the record this was done after my suggestion. Since we are embedding the `PerformanceProfile` struct, and we can pass the new type by pointer like we do for `PerformanceProfile`, this seems ok. I don't really mind moving to the pointer - or removing the new type entirely, though.